include/ibeo_core/ibeo_core.h

```diff
@@ -405,6 +428,28 @@ class ScanPoint2208
     const uint16_t& offset);
 };
 
+class ScanPoint2209
+{
+public:
+  float x_position;
+  float y_position;
+  float z_position;
+  float echo_width;
+  uint8_t device_id;
+  uint8_t layer;
+  uint8_t echo;
+  uint32_t time_offset;
+  bool ground;
+  bool dirt;
+  bool precipitation;
```

Comment:
According to the Ibeo Specification 1.51, there are a few more options here:

0x0001: ground
0x0002: dirt
0x0004: rain/snow/spray/fog/...
0x0008: flag road marking
0x0080: flag curbstone
0x0088: flag guard rail
0x1000: transparent

Would you mind adding them (and parsing them)?